resources/testdriver.js

```diff
@@ -118,9 +126,21 @@
         /**
          * Triggers a user-initiated click
          *
-         * This matches the behaviour of the {@link
-         * https://w3c.github.io/webdriver/#element-click|WebDriver
-         * Element Click command}.
+         * If ``element`` isn't inside the
```

Comment:
```suggestion
         * If `element` isn't inside the
```

Right? Just below it's a single backquote.

Reply:
No, the opposite. The way this works is that even though the documentation file is markdown, the JSDoc ends up in an `eval_rst` block and is interpreted as ReST. That uses the double backtick for inline literals (c.f. [the docs](https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup)).

We also end up using inline ReST for all the parts of JSDoc that sphinx-js doesn't support (which is quite a lot of it).